src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 206px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+}
+
+.card__name {
+  color: $mainAccent;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: $blue-accent;
+  border: 1px solid transparent;
+  color: #fff;
+  &:hover {
+    background-color: #fff;
+    border-color: $blue-accent;
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  &__number {
+    color: $mainAccent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+  &__text {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+}
+
+.product-code {
+  color: $secondary;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.review__text {
+  color: $mainAccent;
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+}
+
+.link {
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  &:hover {
+    color: $blue-accent;
+  }
+}
```

Comment:
Look how it works on hover, fix it please, if you have some problems with it write to chat
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/179f9503-cf18-4545-89de-f75c35fa0775)